backend/app.py

```diff
@@ -53,26 +52,50 @@ def verify():
     # salesforce query based on the email, firstname & lastname
     result = sf.query(format_soql("SELECT Id, Email FROM Contact WHERE (email = {email_value} AND name={full_name})", email_value=email, full_name=name))
 
-    print(result["totalSize"] == 1)
+    # print(result["totalSize"] == 1)
 
     if (result["totalSize"] == 1):
         return {"verified": bool(1)} # true
 	
-	
     return {"verified": bool(0)} # false
 
 @app.route("/calculateProgressBar")
```

Comment:
Rename the endpoint to something more specific to the Home Screen. This endpoint will only return the total outcomes of the pods. Potentially name it "/calcProgressHomeScreen"